src/java.desktop/share/classes/javax/swing/JComponent.java

```diff
@@ -657,6 +657,7 @@ public JComponent() {
 
     /**
      * Resets the UI property to a value from the current look and feel.
```

Comment:
Shall it rather use plural?
```suggestion
     * Resets the UI properties to values from the current look and feel.
```
There are usually several properties that get updated, *all the properties* in fact.

Reply:
I see that all subclass of JComponent implementing updateUI() cites this sentence so changing only here may not be proper unless we change it everywhere, so I guess let it be as it is.

Reply:
Agreed.